### ISSUE TITLE
Project Generation capitalization inconsistency

### ISSUE THREAD

**Issue by SmithRWORNL** [eclipse/ice#189]
In the ICE Item Project Creation Wizard, if the project name is set to include capital letters, the project will have build errors due to inconsistent capitalization. The manifest and java files will include capital letters in their package names, but the package names themselves are all in lower case.

The wizard should either apply a single capitalization scheme or prevent the user from inputting capital letters in the project name.

**Comment by arbennett** [comment]
I just tried this and it seems to have worked fine.  Could you provide more info about how you made this happen?  Specifically it would be helpful to know what you have input on each of the wizard pages.

**Comment by SmithRWORNL** [comment]
Here's what I had in the wizard:

![projectgenerationwizard1](https://cloud.githubusercontent.com/assets/12715269/14604610/84f55de4-0542-11e6-968f-01dfc18b8599.png)
![projectgenerationwizard2](https://cloud.githubusercontent.com/assets/12715269/14604614/87b48636-0542-11e6-9456-aedfed36e611.png)
![projectgenerationwizard3](https://cloud.githubusercontent.com/assets/12715269/14604618/8b33fa58-0542-11e6-9c85-72769f939454.png)

And here's the result I got:
![projectgenerationwizard4](https://cloud.githubusercontent.com/assets/12715269/14604627/9ad28c54-0542-11e6-8a88-c3336028a418.png)

I got the same thing on both Windows and Mac. On Linux, things worked fine, because capitalization was correct for the package names (ie nameWithCapitalLetters.launcher instead of namewithcapitalletters.launcher.)

**Comment by arbennett** [comment]
This also works for me on Linux.  I'm firing up a Windows VM to test this now, thanks.

**Comment by arbennett** [comment]
![virtualbox_windows_18_04_2016_12_00_21](https://cloud.githubusercontent.com/assets/7518706/14616014/563c2986-055d-11e6-898b-763b60225b5f.png)

On Windows 10 with the `unstable-nightly` 32 bit version of ICE I am not able to reproduce this bug.  Have you tried a more recent build?

**Comment by SmithRWORNL** [comment]
I'm still getting the bug on Windows 7 Enterprise Service Pack 1 with the unstable-nightly 64 bit version of ICE.

**Comment by erwindl0** [comment]
I also get the issue : 
- windows 7 professional svc pack 1
- ice unstable-nightly 64 bit from http://eclipseice.ornl.gov/downloads/ice/unstable-nightly/20160418/

**Comment by arbennett** [comment]
Thanks Erwin.  I just tried it again with the 64 bit version of ICE on Windows 10 and still don't experience the issue.  I'm using Java 1.8.0_77 for reference; I've tried both 32 and 64 bit versions of Java as well.

As per Robert's email to the list the following setups were found to experience the bug:
* Windows 7 Enterprise Service Pack 1, Java 1.8.0_45-b15, using the unstable nightly from 4/18/2016
* Windows 7 Enterprise Service Pack 1, Java 1.8.0_71-b15, using the unstable nightly from today.
* Mac OSX 10.9.5, Java 1.8.0_77-b03, using the unstable nightly from today.
